lib/Remotery.c

```diff
@@ -495,7 +495,7 @@ static void AtomicSub(rmtS32 volatile* value, rmtS32 sub)
 // Compiler write fences
 static void WriteFence()
 {
-#if defined(RMT_PLATFORM_WINDOWS) && !defined(__MINGW32__)
+#if defined(RMT_PLATFORM_WINDOWS) && !defined(__MINGW32__) && !defined(__clang__)
```

Comment:
This is the minimal change guaranteed not to rock the boat for other platforms but it would probably be better if the clang predicate was the first one. You would then not need to both check for the absence and existence of clang.